src/chapters/protocol.xml

```diff
@@ -57,29 +57,90 @@
             <programlisting>6.select,3.vnc;</programlisting>
         </informalexample>
         <para>After receiving the "select" instruction, the server will load the associated client
-            support and respond with a list of accepted parameter names using an "args"
-            instruction:</para>
+            support and respond with its protocol version and a list of accepted parameter names
+            using an "args" instruction:</para>
         <informalexample>
-            <programlisting>4.args,8.hostname,4.port,8.password,13.swap-red-blue,9.read-only;</programlisting>
+            <programlisting>4.args,13.VERSION_1_1_0,8.hostname,4.port,8.password,13.swap-red-blue,9.read-only;</programlisting>
         </informalexample>
+        <para>The protocol version is used to negotiate compatibility between differing
+            versions of client and server, allowing the two sides to negotiate the highest
+            supported version and enable or disable features associated with that version.
+            Older versions of the Guacamole Client that do not support this instruction
+            will silently ignore it as an empty connection parameter.  Valid protocol versions
```

Comment:
Valid protocol versions (and their general format) should probably be documented with the instruction in the protocol reference. A good example of this might be the `ack` instruction, which also documents all legal status codes:

https://github.com/apache/guacamole-manual/blob/33e653ccf99cbb7b978719b6ec2cc07a0e8bed30/src/references/instructions/server/stream/ack.xml#L32-L225

Reply:
Added a table with the versions.